test/TestClient.ts

```diff
@@ -1,41 +0,0 @@
-import { Signer } from 'ethers'
```

Comment:
Good catch. Think this was being ignored because it didn't have the `.test.ts` suffix